Laptops/trackpad.md

```diff
@@ -6,9 +6,9 @@
 	
 ## What this SSDT does
 
-This SSDT is used to force enable our GPI0 for VoodooI2C to connect onto.
+A big part of fixing I2C trackpads is enabling them within ACPI. For VoodooI2C to work, GPI0 needs to be enabled, as well as the Trackpad and I2C bus. The latter two devices are commonly disabled behind an OS check for Windows which need to be patched to work with macOS as well.
```

Comment:
As above, maybe mention that you must use the prebuilt option for installs since manual requires macOS installed

Reply:
May be best to just recommend using a mouse for install?